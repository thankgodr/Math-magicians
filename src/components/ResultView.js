import React from 'react';
import PropTypes from 'prop-types';

export default class ResultView extends React.Component {
  render() {
    return (
            <div className="col-md-12 resultview">
              <p className='result'>{this.props.result}</p>
            </div>
    );
  }
}

ResultView.defaultProps = {
  result: 0,
};

ResultView.propTypes = {
  result: PropTypes.number,
};