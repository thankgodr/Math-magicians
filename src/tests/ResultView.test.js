import renderer from 'react-test-renderer';
import ResultView from '../components/ResultView';

describe('Testing snapshot of resultView', () => {
  test('Snapshot of result view', () => {
    const elem = renderer.create(<ResultView />).toJSON();
    expect(elem).toMatchSnapshot();
  });
});