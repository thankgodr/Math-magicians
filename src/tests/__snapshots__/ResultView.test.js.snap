// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing snapshot of resultView Snapshot of result view 1`] = `
<div
  className="col-md-12 resultview"
>
  <p
    className="result"
    id="resultview"
  >
    0
  </p>
</div>
`;
